pyproject.toml

```diff
@@ -6,6 +6,59 @@ requires = [
   "wheel",
 ]
 
+[project]
+name = "windrose"
+description = "Python Matplotlib, Numpy library to manage wind data, draw windrose (also known as a polar rose plot)"
+readme = "README.md"
+license = { text = "BSD-3-Clause OR BCeCILL-B" }
+maintainers = [
+  { name = "Sebastien Celles" },
```

Comment:
@scls19fr there are probably others that should be listed here. I'm not sure who though. Let me know who else we should add to this list in the next iteration.